webapp-django/crashstats/settings/base.py

```diff
@@ -648,7 +642,7 @@ def path(*dirs):
 OIDC_OP_AUTHORIZATION_ENDPOINT = config('OIDC_OP_AUTHORIZATION_ENDPOINT', '')
 OIDC_OP_TOKEN_ENDPOINT = config('OIDC_OP_TOKEN_ENDPOINT', '')
 OIDC_OP_USER_ENDPOINT = config('OIDC_OP_USER_ENDPOINT', '')
-LOGOUT_REDIRECT_URL = '/'
+OIDC_OP_LOGOUT_URL_METHOD = 'crashstats.authentication.utils.redirect_after_logout'
```

Comment:
So I was totally wrong about this; we have to be careful about sanitizing the input coming in from the next parameter. Given that, it'd be easier to just make users go back to the homepage after logging out. My bad.